python/rikai/types/rle.py

```diff
@@ -0,0 +1,65 @@
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from __future__ import annotations
+
+from typing import Tuple
+
+import numpy as np
+
+
+def encode(arr: np.ndarray) -> np.array:
```

Comment:
Do we need to support multi-class semantic segmentation? I think binary is fine for now but am wondering whether we need to leave room in the design for it?

Reply:
So in the coco dataset at least, a labelled segmentation is a combination of `(label:str, mask:nd.array/RLE)`. So it uses label id to identify different classes.